### PR TITLE
Issue 4208 - Fails to inject SecurityContext into Helloworld-CDI2-SE example

### PR DESCRIPTION
**Pull request by pa314159**

### CODE REVIEW

**Comment by jansupol** [comment]
Please update copyright year in the next push

**Comment by pa314159** [comment]
@jansupol, rebase or new commit?

File: inject/cdi2-se/src/main/java/org/glassfish/jersey/inject/cdi/se/bean/SupplierBeanBridge.java [hunk 1]
```diff
@@ -143,4 +143,9 @@ public void destroy(Object instance, CreationalContext context) {
     public Class<? extends Annotation> getScope() {
         return binding.getScope() == null ? Dependent.class : transformScope(binding.getScope());
     }
+
+    @Override
+    public Class<?> getBeanClass() {
```
Comment (jansupol) [inline inject/cdi2-se/src/main/java/org/glassfish/jersey/inject/cdi/se/bean/SupplierBeanBridge.java]: if (binding.getContracts().size() == 1) {
    return binding.getContracts().iterator().next();
} else {
   return super.getBeanClass();
}

**Comment by jansupol** [comment]
rebase is fine, small PR

File: inject/cdi2-se/src/main/java/org/glassfish/jersey/inject/cdi/se/bean/SupplierBeanBridge.java (hunk 1 above)
Comment (pa314159) [reply inject/cdi2-se/src/main/java/org/glassfish/jersey/inject/cdi/se/bean/SupplierBeanBridge.java]: OK, but how is this different than SupplierThreadScopeBean that has the same expression, i.e.

    return (Class<?>) this.binding.getContracts().iterator().next()

Comment (jansupol) [reply inject/cdi2-se/src/main/java/org/glassfish/jersey/inject/cdi/se/bean/SupplierBeanBridge.java]: Right, but better safe than NPE

**Pull request merged**